.github/workflows/ci.yaml

```diff
@@ -133,8 +136,10 @@ jobs:
       - name: Update Nix Flake SRI Hash
         run: ./scripts/update-flake.sh
 
-      - name: Ensure No Changes
-        run: git diff --exit-code
+      - uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          # Allows dependabot to still rebase!
+          commit_message: "[dependabot skip] Update Nix Flake SRI Hash"
```

Comment:
@kylecarbs 

I think we should only do this if the author is @dependabot
For other PRs, this will cause drift, and the author will have to pull/push to synchronize. Also, the message `"[dependabot skip] Update Nix Flake SRI Hash"` doesn't look very nice on a human-created PR.